docs/configuration.md

```diff
@@ -211,4 +211,55 @@ telemetries: [
         }
     ],
 ]
-```
\ No newline at end of file
+```
+
+## Time to Interactive (TTI)
+
+#### What is Time to interactive (TTI)?
+
+Time to interactive (TTI) is a measure of how long a page takes until it is interactive. AWS RUM Web Client provides a plugin you can use to record this measurement. 
+
+#### How is TTI measured by the AWS RUM web client? 
```

Comment:
suggestion: Omit this section's information. Because this file focuses on documenting how to configure the web client, if we want to document how TTI is defined, I think we should do it elsewhere.